File: app/src/main/java/io/github/projectbukirin/palaylab/DiseasesViewPager.java
package io.github.projectbukirin.palaylab;


import android.content.Context;
import androidx.annotation.NonNull;
import androidx.viewpager.widget.PagerAdapter;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;

public class DiseasesViewPager extends PagerAdapter {
    private Context mcontext;
    private int[] images=new int[] {R.drawable.bacterialblight, R.drawable.bacterialblight,R.drawable.bacterialblight,R.drawable.bacterialblight,R.drawable.bacterialblight};

    DiseasesViewPager(Context context){
        mcontext=context;
    }
    @Override
    public int getCount(){
        return images.length;
    }
    @Override
    public boolean isViewFromObject(View view, Object object){
        return view == object;
    }

    @NonNull
    @Override
    public Object instantiateItem(ViewGroup container, int position) {
        ImageView imageView = new ImageView(mcontext);
        imageView.setScaleType(ImageView.ScaleType.CENTER_CROP);
        imageView.setImageResource(images[position]);
        container.addView(imageView,0);
        return imageView;
    }

    @Override
    public void destroyItem(ViewGroup container, int position, Object object) {
        container.removeView((ImageView)object);
    }

    public static class DiseasesViewPager2 extends PagerAdapter {
        private Context mcontext;
        private int[] images=new int[] {R.drawable.riceblast, R.drawable.riceblast,R.drawable.riceblast,R.drawable.riceblast,R.drawable.riceblast};

        DiseasesViewPager2(Context context){
            mcontext=context;
        }
        @Override
        public int getCount(){
            return images.length;
        }
        @Override
        public boolean isViewFromObject(View view, Object object){
            return view == object;
        }

        @NonNull
        @Override
        public Object instantiateItem(ViewGroup container, int position) {
            ImageView imageView = new ImageView(mcontext);
            imageView.setScaleType(ImageView.ScaleType.CENTER_CROP);
            imageView.setImageResource(images[position]);
            container.addView(imageView,0);
            return imageView;
        }

        @Override
        public void destroyItem(ViewGroup container, int position, Object object) {
            container.removeView((ImageView)object);
        }
    }


    public static class DiseasesViewPager3 extends PagerAdapter {
        private Context mcontext;
        private int[] images=new int[] {R.drawable.sheathblight, R.drawable.sheathblight,R.drawable.sheathblight,R.drawable.sheathblight,R.drawable.sheathblight};

        DiseasesViewPager3(Context context){
            mcontext=context;
        }
        @Override
        public int getCount(){
            return images.length;
        }
        @Override
        public boolean isViewFromObject(View view, Object object){
            return view == object;
        }

        @NonNull
        @Override
        public Object instantiateItem(ViewGroup container, int position) {
            ImageView imageView = new ImageView(mcontext);
            imageView.setScaleType(ImageView.ScaleType.CENTER_CROP);
            imageView.setImageResource(images[position]);
            container.addView(imageView,0);
            return imageView;
        }

        @Override
        public void destroyItem(ViewGroup container, int position, Object object) {
            container.removeView((ImageView)object);
        }
    }


    public static class DiseasesViewPager4 extends PagerAdapter {
        private Context mcontext;
        private int[] images=new int[] {R.drawable.tungro, R.drawable.tungro,R.drawable.tungro,R.drawable.tungro,R.drawable.tungro};

        DiseasesViewPager4(Context context){
            mcontext=context;
        }
        @Override
        public int getCount(){
            return images.length;
        }
        @Override
        public boolean isViewFromObject(View view, Object object){
            return view == object;
        }

        @NonNull
        @Override
        public Object instantiateItem(ViewGroup container, int position) {
            ImageView imageView = new ImageView(mcontext);
            imageView.setScaleType(ImageView.ScaleType.CENTER_CROP);
            imageView.setImageResource(images[position]);
            container.addView(imageView,0);
            return imageView;
        }

        @Override
        public void destroyItem(ViewGroup container, int position, Object object) {
            container.removeView((ImageView)object);
        }
    }

}

File: app/src/main/kotlin/xyz/marcreyes/palaylab/app/disease/DiseasesViewPager.kt
package xyz.marcreyes.palaylab.app.disease

import android.content.Context
import android.view.View
import android.view.ViewGroup
import android.widget.ImageView
import androidx.viewpager.widget.PagerAdapter
import xyz.marcreyes.palaylab.app.R

class DiseasesViewPager internal constructor(private val mcontext: Context) : PagerAdapter() {
    private val images = intArrayOf(R.drawable.bacterialblight, R.drawable.bacterialblight, R.drawable.bacterialblight, R.drawable.bacterialblight, R.drawable.bacterialblight)
    override fun getCount(): Int {
        return images.size
    }

    override fun isViewFromObject(view: View, `object`: Any): Boolean {
        return view === `object`
    }

    override fun instantiateItem(container: ViewGroup, position: Int): Any {
        val imageView = ImageView(mcontext)
        imageView.scaleType = ImageView.ScaleType.CENTER_CROP
        imageView.setImageResource(images[position])
        container.addView(imageView, 0)
        return imageView
    }

    override fun destroyItem(container: ViewGroup, position: Int, `object`: Any) {
        container.removeView(`object` as ImageView)
    }

    class DiseasesViewPager2 internal constructor(private val mcontext: Context) : PagerAdapter() {
        private val images = intArrayOf(R.drawable.riceblast, R.drawable.riceblast, R.drawable.riceblast, R.drawable.riceblast, R.drawable.riceblast)
        override fun getCount(): Int {
            return images.size
        }

        override fun isViewFromObject(view: View, `object`: Any): Boolean {
            return view === `object`
        }

        override fun instantiateItem(container: ViewGroup, position: Int): Any {
            val imageView = ImageView(mcontext)
            imageView.scaleType = ImageView.ScaleType.CENTER_CROP
            imageView.setImageResource(images[position])
            container.addView(imageView, 0)
            return imageView
        }

        override fun destroyItem(container: ViewGroup, position: Int, `object`: Any) {
            container.removeView(`object` as ImageView)
        }
    }

    class DiseasesViewPager3 internal constructor(private val mcontext: Context) : PagerAdapter() {
        private val images = intArrayOf(R.drawable.sheathblight, R.drawable.sheathblight, R.drawable.sheathblight, R.drawable.sheathblight, R.drawable.sheathblight)
        override fun getCount(): Int {
            return images.size
        }

        override fun isViewFromObject(view: View, `object`: Any): Boolean {
            return view === `object`
        }

        override fun instantiateItem(container: ViewGroup, position: Int): Any {
            val imageView = ImageView(mcontext)
            imageView.scaleType = ImageView.ScaleType.CENTER_CROP
            imageView.setImageResource(images[position])
            container.addView(imageView, 0)
            return imageView
        }

        override fun destroyItem(container: ViewGroup, position: Int, `object`: Any) {
            container.removeView(`object` as ImageView)
        }
    }

    class DiseasesViewPager4 internal constructor(private val mcontext: Context) : PagerAdapter() {
        private val images = intArrayOf(R.drawable.tungro, R.drawable.tungro, R.drawable.tungro, R.drawable.tungro, R.drawable.tungro)
        override fun getCount(): Int {
            return images.size
        }

        override fun isViewFromObject(view: View, `object`: Any): Boolean {
            return view === `object`
        }

        override fun instantiateItem(container: ViewGroup, position: Int): Any {
            val imageView = ImageView(mcontext)
            imageView.scaleType = ImageView.ScaleType.CENTER_CROP
            imageView.setImageResource(images[position])
            container.addView(imageView, 0)
            return imageView
        }

        override fun destroyItem(container: ViewGroup, position: Int, `object`: Any) {
            container.removeView(`object` as ImageView)
        }
    }
}